### ISSUE TITLE
Fix QA report issues

### ISSUE THREAD

**Issue by regelkraut** [HL7Austria/HL7-AT-FHIR-Core-R4#72]
**Real Submitter**
@regelkraut 

**Resource(s)/Profiles incl. Version**
The whole HL7 Austria FHIR Core IG in its current version.

**Links**
[HL7 Austria FHIR Core IG STU2](https://fhir.hl7.at/r4-core-main/index.html)

**Priority**
Major

**Describe the bug**
Since the last couple of updates of the IG Publisher the number of complaints and errors in the QA report of the IG increased rather significantly.

**To Reproduce**
1. Just check the QA report of a current build with the newest version of the IG Publisher

**Expected behavior**
To resolce all (real) errors in the QA report.

**Attachments**
n/a

**Additional context**
n/a

**Comment by alackerbauer** [comment]
I added an ignoreWarnings section to remove the errors from the list that we are already aware of (see PR https://github.com/HL7Austria/HL7-AT-FHIR-Core-R4/pull/73). 

In terms of the broken links: I went through all files in the HL7 AT Core R4 IG and IMO there is nothing wrong with them. What is causing this message much more is an outdated version of copied files in our Austrian template IG (e.g. https://github.com/HL7Austria/at-fhir-ig-template/blob/master/layouts/layout-profile.html). I haven't updated those files since I wanted to understand first, if there is any need to have a local copy of them in the first place and if there is any reason why we are not inheriting our Austrian template from the fhir.base.template?

**Comment by regelkraut** [comment]
Hi @alackerbauer,
good question, the interesting thing is that the dependency checks on the qa report page of our IG say, that there is a relationship between our HL7 AT template and the FHIR base template:
![TemplateDependencyt](https://github.com/HL7Austria/HL7-AT-FHIR-Core-R4/assets/39377217/d40d6b68-e13d-432c-81a7-10aecf436d57)
(btw. the current version of our HL7 AT template seems to be 0.3.0 and not 0.1.0 as we have currently in the ig.ini file of our main build, because of that the IG Publisher kept complaining while I was trying to create a local build. After the switch to 0.3.0 it worked again, but the errors in the qa report remained.)
We should ask @aschuler84 who created the HL7 AT template if there is a special reason for that.

**Comment by alackerbauer** [comment]
Hi @regelkraut!
Thanks for your point - I ran into a similar issue with 0.1.0, I tried to update to _current_ but there seems to be an issue with the [published template](http://fhir.org/templates/hl7.at.fhir.template/history.html) as there is no entry when you click on current. So I hard-coded the version to 0.3.0 locally for now.

Yes, we do inherit from the current base template - see [here](https://github.com/HL7Austria/at-fhir-ig-template/blob/master/package/package.json#L9), but then override the layout files in our local template version. The template file is still from November 2021, and back then, it was almost identical to the layout file from the base template (see diff [here](https://www.diffchecker.com/B92mMkxV/) - I do not know if that diff is relevant for us to have. If not, I suggest just deleting the layout files from the Austrian template if that is possible. If they are relevant, I suggest taking the latest layout versions from the base template and adjusting them with the [diff](https://www.diffchecker.com/B92mMkxV/) and hopefully this would fix the broken links.

One thing to note: the Austrian template 0.1.0 didn't yet use the current base template but a specified version, so based on what version of the Austrian template you use, you might also have a different base template in your build.
